x-pack/test/observability_ai_assistant_api_integration/common/config.ts

```diff
@@ -68,8 +75,8 @@ export function createObservabilityAIAssistantAPIConfig({
         getApmSynthtraceEsClient(context, apmSynthtraceKibanaClient),
       observabilityAIAssistantAPIClient: async () => {
         return {
-          readUser: await getObservabilityAIAssistantAPIClient(kibanaServerUrl),
-          writeUser: await getObservabilityAIAssistantAPIClient(kibanaServerUrl),
+          readUser: await getObservabilityAIAssistantAPIClient(kibanaServer, viewerUser.username),
+          writeUser: await getObservabilityAIAssistantAPIClient(kibanaServer, editorUser.username),
```

Comment:
Maybe we should align on read/write **or** view/edit instead of mixing?

Reply:
Yes, I thought about that but didn't want to pollute this PR since it would update every test. Not a big deal those so happy to do it,

Reply:
I don't have a strong opinion on it